sdks/go/model/scg.go

```diff
@@ -32,7 +32,7 @@ type SCGContainerCall struct {
 
 type SCGContainerCallImage struct {
 	Src       *string       `json:"src,omitempty"`
-	Ref       string        `json:"ref"`
+	Ref       string        `json:"ref,omitempty"`
```

Comment:
I think Ref's type also needs to be a pointer so that it can be an empty value. See https://stackoverflow.com/questions/38486564/unmarshal-marshal-json-with-int-set-to-0-does-not-seem-to-work